activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,57 @@
+require 'active_support/time'
+
+module ActiveSupport
+  class Claims
```

Comment:
Also, we would not want people to have a look at ```LegacyMessageVerifier``` either, right ? So, a  ```:nodoc:``` for that too would be needed.